ceph-dev-setup/build/build

```diff
@@ -43,25 +43,45 @@ echo "Running submodule update ..."
 git submodule update --init
 
 # Flavor Builds support
-# TODO: enable CMAKE
-#export CEPH_EXTRA_CMAKE_ARGS
 
-FLAVOR_FLAGS="--without-cryptopp"
+#cryptopp is not default for CMake
+AUTOCONF_FLAVOR_FLAGS="--without-cryptopp"
 
 if [ "${FLAVOR}" == "notcmalloc" ]
 then
-    echo "Detected notcmalloc flavor: will use flag: --without-tcmalloc"
-    FLAVOR_FLAGS="$FLAVOR_FLAGS --without-tcmalloc"
-    #export CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=libc"
+    echo "Detected notcmalloc flavor: will use flag: --without-tcmalloc or -DALLOCATOR=libc"
+    AUTOCONF_FLAVOR_FLAGS="$AUTOCONF_FLAVOR_FLAGS --without-tcmalloc"
+    export CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=libc"
 else
-    FLAVOR_FLAGS="$FLAVOR_FLAGS --with-tcmalloc"
-    #export CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=tcmalloc"
+    AUTOCONF_FLAVOR_FLAGS="$AUTOCONF_FLAVOR_FLAGS --with-tcmalloc"
+    export CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=tcmalloc"
 fi
 
-if [ "${FLAVOR}" == "blkin" ]
-then
-    echo "Detected bklin flavor: will use flag: --with-blkin"
-    FLAVOR_FLAGS="$FLAVOR_FLAGS --with-blkin"
+
+# make-dist was renamed in conjuntion with cmake cutover
+if [ -x make-dist ]
```

Comment:
those should be the same. We don't support branches with forward slashes in them because we use a binary store in an HTTP API. Since the "branch" is part of the URL it cannot have a slash in it